wallet/createtx.go

```diff
@@ -354,7 +354,7 @@ func (w *Wallet) addrMgrWithChangeSource(dbtx walletdb.ReadWriteTx,
 
 	// Determine the address type for change addresses of the given account.
 	if changeKeyScope == nil {
-		changeKeyScope = &waddrmgr.KeyScopeBIP0084
+		changeKeyScope = &waddrmgr.KeyScopeBIP0086
```

Comment:
Docs on `addrMgrWithChangeSource` and `txToOutputs` both still indicate P2WPKH will be used for change if keyscope not specified.  Docs aside, I'm uncertain how to handle this in our app where we want all utxos selected for funding, but we expect the change to be P2WPKH.

May optional/variadic arg for `CreateSimpleTx` and `SendOutputs`?  Or maybe differently named exported methods with change type args?

Reply:
> May optional/variadic arg for CreateSimpleTx and SendOutputs?

Leaning towards this option, so then a potential change would be:
  * New optional arg for what the change addr scope should be 
  * If it isn't set, then it'll use the scope selected earlier (either default or specified scope) 
  * if set, then that's used for change addrs too 

Then for our desired use case (on the lnd side), we'd leave the existing arg, and start setting this new change arg.

Reply:
Ok did the the optional arg thing @chappjc, can you check out the latest diff? Want to make sure it still works for y'all's use case. I've also added a test to exercise the feature.